mlir/test/Integration/Dialect/SparseTensor/CPU/sparse_block3d.mlir

```diff
@@ -30,6 +30,10 @@
 // Do the same run, but now with direct IR generation and VLA vectorization.
 // RUN: %if mlir_arm_sve_tests %{ %{compile_sve} | %{run_sve} | FileCheck %s %}
 
+// Make sure that test-bufferization-analysis-only works. This option is useful
```

Comment:
The way it was phrased looked initially like a TODO to me, but then I realized it was simply tested.

Can we rephrase this as

Test that ....